server/src/main/java/keywhiz/service/daos/GroupDAO.java

```diff
@@ -49,7 +55,25 @@ private GroupDAO(DSLContext dslContext, GroupMapper groupMapper, ObjectMapper ma
     this.mapper = mapper;
   }
 
-  public long createGroup(String name, String creator, String description, ImmutableMap<String, String> metadata) {
+  public long createGroup(
+      String name,
+      String creator,
+      String description,
+      ImmutableMap<String, String> metadata) {
+    return createGroup(
+        name,
+        creator,
+        description,
+        metadata,
+        NO_OWNER);
+  }
```

Comment:
As much as I prefer to avoid the proliferation of overloads for backwards compatibility's sake, this is just temporary until we wire up the corresponding Resources (which sit one level higher).